stylesheets/partials/features/_contact-form.scss

```diff
@@ -1,109 +1,105 @@
-.contact-form {
-  background: $gray-lightest;
-  box-shadow: $default-box-shadow;
-  margin-bottom: 30px;
-  padding: 15px 15px 0;
-
-  h3 {
-    font-size: 2.1em;
-  }
+.contact-form-container form {
```

Comment:
Does coveralls include CSS and SCSS in its coverage percentage?  All the net new lines in this PR are CSS rules rather than JS.  Counting that as part of coverage seems counterproductive.

Reply:
no, this was caused by broken tests.